src/pages/workspace/tags/WorkspaceTagsPage.tsx

```diff
@@ -423,140 +426,141 @@ function WorkspaceTagsPage({route}: WorkspaceTagsPageProps) {
     const selectionModeHeader = selectionMode?.isEnabled && shouldUseNarrowLayout;
 
     return (
-        <AccessOrNotFoundWrapper
-            policyID={policyID}
-            accessVariants={[CONST.POLICY.ACCESS_VARIANTS.ADMIN, CONST.POLICY.ACCESS_VARIANTS.PAID]}
-            featureName={CONST.POLICY.MORE_FEATURES.ARE_TAGS_ENABLED}
-        >
-            <ScreenWrapper
-                enableEdgeToEdgeBottomSafeAreaPadding
-                style={[styles.defaultModalContainer]}
-                testID={WorkspaceTagsPage.displayName}
-                shouldShowOfflineIndicatorInWideScreen
-                offlineIndicatorStyle={styles.mtAuto}
+        <>
```

Comment:
Why is this extra view needed?

Reply:
Closing animation of the modal was broken when the `ConfirmModal` was nested inside `AccessOrNotFoundWrapper`, it was because it was unmounted for one frame and the animation was interrupted this way. 

Fixed it right before [this](https://github.com/Expensify/App/pull/60023#issuecomment-2828473555) comment.

Reply:
Ok thanks!